easybuild/easyconfigs/p/PAGIT/PAGIT-V1.64bit.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'PAGIT'
+version = 'V1'
+versionsuffix = '.64bit'
+
+homepage = 'http://www.sanger.ac.uk/science/tools/pagit'
+description = """PAGIT addresses the need for software to generate high quality draft genomes"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/pagit/']
+sources = ['%(name)s.%(version)s%(versionsuffix)s.tgz']
+
+osdependencies = ['tcsh']
+
+modextrapaths = {
+       'PATH': "bin:$root/PAGIT/bin:$root/PAGIT/bin/pileup_v0.5/:$root/PAGIT/bin/pileup_v0.5/ssaha2:$root/PAGIT/IMAGE/:$root/PAGIT/ABACAS:$root/PAGIT/ICORN:$root/PAGIT/RATT",
+       'PERL5LIB': 'PAGIT/lib',
+}
+
+modextravars = {
+    'PAGIT_HOME':'$root/PAGIT',
```

Comment:
thanks for the tip, I was not aware of this. I will need to update many of my easyconfigs because now I am migrating to lua syntax

btw, I noticed that `$root` is being used in some easyconfig in the main release:

```
$ grep -R "\$root" *|grep -v .patch
easybuild/easyconfigs/m/MView/MView-1.57-goolf-1.4.10-Perl-5.16.3.eb:    'PERL5LIB': '$root/lib'
easybuild/easyconfigs/m/MPJ-Express/MPJ-Express-0.44-goolf-1.4.10-Java-1.7.0_75.eb:    'MPJ_HOME': '$root',  # MPJ_HOME points to install directory
easybuild/easyconfigs/m/MPJ-Express/MPJ-Express-0.44-foss-2016a-Java-1.8.0_92.eb:    'MPJ_HOME': '$root',  # MPJ_HOME points to install directory
easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb:    'WEKAINSTALL': '$root',
easybuild/easyconfigs/w/WEKA/WEKA-3.7.0-Java-1.7.0_80.eb:    'WEKAINSTALL': '$root',
easybuild/easyconfigs/a/ALPS/ALPS-2.3.0-foss-2016b-Python-3.5.2.eb:    'ALPS_ROOT': '$root',
easybuild/easyconfigs/a/ALPS/ALPS-2.3.0-foss-2016b-Python-2.7.12.eb:    'ALPS_ROOT': '$root',
```